web/themes/custom/lex/scss/_variables.scss

```diff
@@ -1,6 +1,6 @@
 $breakpoints: (
   xs: 576px,
-  sm: 768px,
+  sm: 800px,
```

Comment:
@TaylorAkin 768px is usually the accepted breakpoint width for small screens. Kinda going against the grain on this one.